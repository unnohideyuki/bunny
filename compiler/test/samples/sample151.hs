main = do
  print $ LT == LT
  print $ LT == EQ
  print $ LT == GT
  print $ EQ == LT
  print $ EQ == EQ
  print $ EQ == GT
  print $ GT == LT
  print $ GT == EQ
  print $ GT == GT

  print $ LT /= LT
  print $ LT /= EQ
  print $ LT /= GT
  print $ EQ /= LT
  print $ EQ /= EQ
  print $ EQ /= GT
  print $ GT /= LT
  print $ GT /= EQ
  print $ GT /= GT
