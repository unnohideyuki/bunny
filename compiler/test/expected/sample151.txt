True
False
False
False
True
False
False
False
True
False
True
True
True
False
True
True
True
False
